web-server/v0.4/src/utils/parse.js

```diff
@@ -300,3 +316,74 @@ export const generateIterationClusters = (config, iterations) => {
 
   return parseClusteredIterations(primaryMetricIterations, clusterLabels, selectedConfig);
 };
+
+export const getComparisonColumn = (record, iterationNo) => {
+  const children = [];
+  for (let i = 0; i < iterationNo; i += 1) {
+    children.push({
+      title: `Iteration - ${i}`,
+      dataIndex: '0',
+      key: '0',
+      children: [
+        {
+          title: 'Name',
+          dataIndex: `name${i}`,
+          key: i,
+          width: 250,
+        },
+        {
+          title: 'Mean',
+          dataIndex: i,
+          key: i,
+          width: 250,
+        },
+        {
+          title: 'Percent',
+          dataIndex: `percent${i}`,
+          key: `percent${i}`,
+          width: 0,
+          render: text => {
+            if (text > 0) {
+              return (
+                <div style={{ textAlign: 'right' }}>
+                  {`${text}%`}
+                  <Icon
+                    type="caret-up"
+                    theme="filled"
+                    style={{ marginLeft: '3%', color: 'green' }}
+                  />
+                </div>
+              );
+            }
+            if (text === 0) {
+              return `${text}%`;
+            }
+            if (text === undefined) {
```

Comment:
Can this edge case be caught during the parsing phase so we don't need to account for this during the rendering phase?